src/rules/czech.toml

```diff
@@ -0,0 +1,16 @@
+allowed_symbols_regex="[A-Za-zěščřžýáíéóďťňĚŠČŘŽÝÁÍÉÓĎŤŇäöüÄÖÜ‚–\\. ]"
```

Comment:
would it make sense to allow quotation symbols? Would that change anything in terms of resulting total sentences?

Reply:
A run with just adding double quotes resulted in 5k more sentences, I will check on their error rate

Reply:
Added in the last commit